Team2 tund20 implement extent report

tund-hcmue opened the pull request:
- create new class ExtentManager
- create new class ExtentTestManager
- create new class ReportListener
- add new method waitForElementToBeClickable in class WebKeyword
- Edit keyword setValueForElement used waitForElementToBeClickable

tund-hcmue committed:
create extent report apply for project:
- create new class ExtentManager
- create new class ExtentTestManager
- create new class ReportListener

tund-hcmue committed:
Merge branch 'team2-tund20-implement-test-case-about-search-location-module' into team2-tund20-implement-extent-report

tund-hcmue committed:
- update extent report
- update keyword setValueForElement in class WebKeyword

TrangDao25399 commented on demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java:
```diff
@@ -39,7 +39,7 @@ public WebKeyword(WebDriver driver) {
      * @return WebKeyword to set value for element
      */
     public WebKeyword setValueForElement(WebElement webElement, chooseTypeOfSelect type, String value){
```
rename to setValueForSelectElement or setValueForDdlElement

TrangDao25399 commented on demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java:
```diff
@@ -39,7 +39,7 @@ public WebKeyword(WebDriver driver) {
      * @return WebKeyword to set value for element
      */
     public WebKeyword setValueForElement(WebElement webElement, chooseTypeOfSelect type, String value){
```
I think you should rename to setValueForSelectElement or setValueForDdlElement

tund-hcmue replied:
thanks for feedback, i was rename it.

tund-hcmue committed:
I rename setValueForElement keyword to setValueForSelectElement

TrangDao25399 reviewed (approved):
I think it is good

herrynhat commented:
Good Idea

tund-hcmue committed:
resolve conflict

tund-hcmue committed:
- create CarForSaleSearchLocationPage.java
- create CarForSaleSearchLocationTest.java
- create CarForSaleTestData.java
- create distancedata.json
create zipcodedata.json
